src/pages/resultsView/querySummary/QuerySummary.tsx

```diff
@@ -145,6 +145,7 @@ export default class QuerySummary extends React.Component<{ routingStore:Extende
         return <div style={{marginTop:10}}>
             <QueryAndDownloadTabs onSubmit={this.onSubmit}
                                   showQuickSearchTab={AppConfig.serverConfig.quick_search_enabled}
+                                  onlyShowQueryTab={true}
```

Comment:
@dippindots 	i think since we have one property `showQuickSearchTab`, it's slightly unclear what the combination `onlyShowQueryTab` & `showQuickSearchTab` does. Maybe better to add a prop `showDownloadTab` and set both `showQuickSearchTab` and `showDownloadTab` to false on the resultsviewpage

Reply:
Sure!